README.md

```diff
@@ -149,12 +149,9 @@ The ```user_data``` attribute is entirely custom, however in [wrld-example-app](
 |`phone`|string| a phone number for the poi
 |`web`|string| a website for the poi
 |`description`|string| a description for the poi
-|`highlight`|string| id of the highlight to enable for a poi
+|`highlight`|string\|string[]| id(s) of the highlight(s) to enable for a poi
```

Comment:
Please add a space either side of the `|`

Also, worth changing `array` to `int[]` on the following line for consistency